include/aspect/postprocess/visualization/maximum_shear_stress.h

```diff
@@ -0,0 +1,89 @@
+/*
+  Copyright (C) 2016 - 2017 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_postprocess_visualization_maximum_shear_stress_h
+#define _aspect_postprocess_visualization_maximum_shear_stress_h
+
+#include <aspect/postprocess/visualization.h>
+#include <aspect/simulator_access.h>
+
+#include <deal.II/numerics/data_postprocessor.h>
+
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+    namespace VisualizationPostprocessors
+    {
+      /**
+       * A class that computes a field of horizontal vectors that
+       * represent the direction of maximal horizontal compressive
+       * stress. For an exact definition, see the documentation of
```

Comment:
you'll have to update this -- it's not the compressive, but the shear stress I believe